http/tokenV2/authorized_keys.go

```diff
@@ -82,6 +82,15 @@ func parseAuthorizedKeys(contents []byte) ([]authorizedKey, error) {
 		if err != nil {
 			return nil, fmt.Errorf("unparseable line (%v): %w", line, err)
 		}
+                
+                // Trim whitespace from the comment/username
+                comment = strings.TrimSpace(comment)
+                
+                // Ignore keys without a comment/username
+                if comment == "" {
+                        log.Logger().Warnf("ignoring authorized_keys entry without comment/username: %v", line)
```

Comment:
We have the convention that logs start with a captial (and errors start lowercase, as Go requires)
```suggestion
                        log.Logger().Warnf("Ignoring authorized_keys entry without comment/username: %v", line)
```